napari/layers/tracks/_tests/test_tracks.py

```diff
@@ -197,3 +197,15 @@ def test_tracks_length_change():
     layer.head_length = track_length
     assert layer.head_length == track_length
     assert layer._max_length == track_length
+
+
+def test_track_ids_ordering() -> None:
+    """Check if tracks ids are correctly set to features when given not-sorted tracks."""
+    # track_id, t, y, x
+    data = np.asarray(
+        [[1, 1, 0, 0], [0, 1, 0, 0], [2, 0, 0, 0], [0, 0, 0, 0], [1, 0, 0, 0]]
+    )
+    track_ids = [0, 0, 1, 1, 2]  # track_ids after sorting
```

Comment:
Optional: prefer expressive variable names over implementation comments.

```suggestion
    sorted_track_ids = [0, 0, 1, 1, 2]
```